hypervisor/dm/vpci/vpci.c

```diff
@@ -436,10 +436,16 @@ static const struct cfg_header_perm cfg_hdr_perm = {
 /*
  * @pre offset + bytes < PCI_CFG_HEADER_LENGTH
  */
-static void read_cfg_header(const struct pci_vdev *vdev,
+static int32_t read_cfg_header(const struct pci_vdev *vdev,
 		uint32_t offset, uint32_t bytes, uint32_t *val)
 {
-	if (vbar_access(vdev, offset)) {
+	int32_t ret;
```

Comment:
I prefer using "int32_t ret = 0" here.

Reply:
Thanks for pointing out. 
It is changed.